github/compliance/issue.py

```diff
@@ -299,16 +300,17 @@ def close_issue_if_present(
             issue_type=issue_type,
             known_issues=known_issues,
             state='open',
+            digest_label_prefix=digest_label_prefix,
         )
     )
     open_issues: tuple[github3.issues.ShortIssue]
 
     logger.info(f'{len(open_issues)=} found for closing {open_issues=}')
 
     if (issues_count := len(open_issues)) > 1:
-        logger.warning(f'more than one open issue found for {_target_name(target)=}')
+        logger.warning(f'more than one open issue found for {target_name(target)=}')
```

Comment:
should remain private

Reply:
lets discuss here https://github.com/gardener/cc-utils/pull/775/files#r1025169735